### ISSUE TITLE
[Bug] failure of webpack-dev-server to spawn a process seems affected by PnP

### ISSUE THREAD

**Issue by fredmenez** [yarnpkg/berry#1082]
**Describe the bug**

A typical usage of webpack-dev-server is expected to open a browser tab. With berrry in PnP mode  
the repro scenario (see below) outputs the following error but works fine if `nodeLinker: node-modules` is set in `.yarnrc.yml`

```
internal/child_process.js:373
    throw errnoException(err, 'spawn');
    ^

Error: spawn ENOTDIR
    at ChildProcess.spawn (internal/child_process.js:373:11)
    at Object.spawn (child_process.js:549:9)
    at module.exports (/work/yarn-pnp-webpacker-server-bug/.yarn/cache/opn-npm-5.5.0-9a97e03147-2.zip/node_modules/opn/index.js:77:26)
    at /work/yarn-pnp-webpacker-server-bug/.yarn/$$virtual/webpack-dev-server-virtual-32a150ad87/0/cache/webpack-dev-server-npm-3.10.3-c02c5d42a6-2.zip/node_modules/webpack-dev-server/lib/utils/runOpen.js:25:14
    at Array.map (<anonymous>)
    at runOpen (/work/yarn-pnp-webpacker-server-bug/.yarn/$$virtual/webpack-dev-server-virtual-32a150ad87/0/cache/webpack-dev-server-npm-3.10.3-c02c5d42a6-2.zip/node_modules/webpack-dev-server/lib/utils/runOpen.js:22:11)
    at status (/work/yarn-pnp-webpacker-server-bug/.yarn/$$virtual/webpack-dev-server-virtual-32a150ad87/0/cache/webpack-dev-server-npm-3.10.3-c02c5d42a6-2.zip/node_modules/webpack-dev-server/lib/utils/status.js:57:5)
    at Server.showStatus (/work/yarn-pnp-webpacker-server-bug/.yarn/$$virtual/webpack-dev-server-virtual-32a150ad87/0/cache/webpack-dev-server-npm-3.10.3-c02c5d42a6-2.zip/node_modules/webpack-dev-server/lib/Server.js:749:5)
    at Server.<anonymous> (/work/yarn-pnp-webpacker-server-bug/.yarn/$$virtual/webpack-dev-server-virtual-32a150ad87/0/cache/webpack-dev-server-npm-3.10.3-c02c5d42a6-2.zip/node_modules/webpack-dev-server/lib/Server.js:767:12)
    at Object.onceWrapper (events.js:284:20)
```

But it works in non PnP mode by adding`nodeLinker: node-modules` to `.yarnrc.yml`.

**To Reproduce**

Run : 

```
#!/bin/bash

echo '{
    "name": "meh",
    "version": "1.0.0",
    "main": "index.js",
    "private": true,
    "scripts": {
        "dev": "webpack-dev-server --open --progress",
        "build": "webpack"
    },
    "devDependencies": {
        "pnp-webpack-plugin": "^1.6.4",
        "webpack": "^4.42.0",
        "webpack-cli": "^3.3.11",
        "webpack-dev-server": "^3.10.3"
    }
}' > package.json

echo '
<!doctype html>
<html>
  <head>
  </head>
  <body>
    <script src="./main.bundle.js"></script>
  </body>
</html>
' > index.html

mkdir src
echo 'alert("woot");' > src/index.js

echo '
const Path = require("path");
const PnPWebpackPlugin = require("pnp-webpack-plugin");

module.exports = {
    mode: "development",

    resolve: {
        plugins: [
            PnPWebpackPlugin
        ],
    },

    resolveLoader: {
        plugins: [
            PnPWebpackPlugin.moduleLoader(module),
        ],
    },
    
    devtool: "inline-source-map",
    
    devServer: {
        contentBase: Path.join(__dirname, "dist")
        , disableHostCheck: true
    },
    
    entry: "./src/index.js",
    
    output: {
        filename: "[name].bundle.js",
        path: Path.resolve(__dirname, "dist"),
    },
};
' > webpack.config.js

yarn set version berry
yarn install

yarn run dev

```

Wasn't aware of sherlock, i'll see if i can update the issue to use it.

**Environment if relevant (please complete the following information):**

 - OS: Linux Mint 19.1
 - Node v12.0.0
 - Yarn version 2.0.0-rc.30

**Comment by fredmenez** [comment]
Issue doesn't occur when package `opn` is unplugged :

```
yarn unplug opn
yarn run dev 
```
➡️ new browser tab is opened as expected

**Comment by fredmenez** [comment]
Closed as duplicate of #856
